uwblueprint/jump-math
# feat: user redirect

## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/User-Redirect-fb13debc0e12438481b0bd3ff764e52c)



## Implementation description
* `Login.tsx`: added logic to redirect user to the correct page (`TeacherPage` or `AdminPage`) depending on role
* `TeacherPage.tsx`, `AdminPage.tsx`: added buttons for logging out, changing password, etc. as described in the Notion ticket
* `Default.tsx`: deleted due to redundancy


## Steps to test
1. Log in as an admin -> verify that you're redirected to the admin page
2. Log in as a teacher -> verify that you're redirected to the teacher page


## What should reviewers focus on?
* Unsure if we should be removing `Default.tsx` entirely - maybe check if user is authenticated and then redirect to login/teacher page/admin page/etc. based on that instead?


## Checklist
